test/postman/inference_data.json

```diff
@@ -202,6 +202,14 @@
         "content-type":"text/plain",
         "expected":"0"
     },
+    {
+      "url":"{{mar_path_mnist_base_cpp}}",
+      "model_name":"mnist_base_cpp",
+      "worker":1,
+      "synchronous":"true",
+      "file":"../cpp/test/resources/torchscript_model/mnist/0_png.pt",
+      "content-type":"binary"
```

Comment:
- I have used `binary` for content-type for testing purposes. Backend will just add the same content-type in the response headers. The more relevant content-type could be `application/zip` as postman is recognizing the .pt file as a zip.
- I am not comfortable doing a bytes comparison since the results on all platforms may not be the same bitwise and our existing tests also account for some `tolerance` between expected and computed results. Bytes comparison might lead to flaky test results.

Reply:
the other option is to probably add a separate integration test under cpp folder in python so that we keep cpp test isolated since we are still in beta.

Reply:
example mnist handler always return 0. you can switch your test to example mnist. Then the verification is quite simple.

Reply:
Added response validation by converting the output torch tensor into an int8 array.